fuzz/fsfuzzer.py

```diff
@@ -90,7 +90,8 @@ def test(self):
         Runs the fsfuzz test suite. By default uses all supported fstypes,
         but you can specify only one by `fstype` param.
         '''
-        process.system("%s %s" % (self._fsfuzz, self._args), sudo=True)
+        process.system("%s %s" % (self._fsfuzz, self._args), sudo=True,
+                       ignore_status=True)
```

Comment:
@PraveenPenguin what is the plan here to pass / fail the test ?

Reply:
@narasimhan-v still i need to figure out  which condition i need to have for fail and it will take some time so thinking i can have this on coming patch as checking exit status of this not really help

Reply:
@PraveenPenguin I guess then we can use ignore_status=True when we come up with that. If we make the test PASS irrespective of the test effect, we will have false PASSES.

Your thoughts ?

Reply:
@narasimhan-v as this is fuzzer so i think should be ok , if you think i will check exit status